vendor/jasmine3-runner.js

```diff
@@ -0,0 +1,22 @@
+const { createRunner } = require("atom-jasmine3-test-runner");
+
+// https://github.com/UziTech/atom-jasmine3-test-runner#api
+module.exports = createRunner({
+  suffix: "-spec-v3",
+  legacySuffix: "-spec",
+  testPackages: [],
+  timeReporter: true,
+  specHelper: true,
+  attachToDOM: true,
+  // Extra Packages
+  customMatchers: true,
+  jasmineFocused: true,
+  jasmineJson: true,
+  jasminePass: true,
+  jasmineShouldFail: true,
+  jasmineTagged: true,
+  mockClock: true,
+  mockLocalStorage: true,
+  profile: true,
+  unspy: true,
```

Comment:
if `specHelper` is `true` you don't need to specify each spec helper it will include them all. If you only want to include some then `specHelper` would be an object with the ones you want set to `true`.

e.g.
```js
  specHelper: {
    attachToDom: true,
    ...
  }
```

```suggestion
  suffix: "-spec-v3",
  legacySuffix: "-spec",
  testPackages: [],
  timeReporter: true,
  specHelper: true,
```

Reply:
Thanks! Good to know. I want to see the result for the previous run before committing this.